UI/window-basic-main.cpp

```diff
@@ -165,9 +165,12 @@ static void AddExtraModulePaths()
 	}
 
 	char base_module_dir[512];
```

Comment:
This should probably use `PATH_MAX` (and aliased from `MAX_PATH` on Windows).

Reply:
I'd consider that to be out-of-scope of this PR, as this is something that would also effect other operating systems (and isn't even modified by this PR).